stdlib/Makefile

```diff
@@ -0,0 +1,34 @@
+default: get
```

Comment:
Could you tell more about this approach v.s. git submodule?

Reply:
see https://github.com/JuliaLang/julia/pull/10743, submodules were used once upon a time for deps and it was pretty awful